exasol/exaslpm/pkg_mgmt/cmd_executor.py

```diff
@@ -0,0 +1,94 @@
+import subprocess
+import sys
+from collections.abc import (
+    Callable,
+    Iterator,
+)
+from typing import Protocol
+
+
+class CommandLogger(Protocol):
+    def info(self, msg: str, **kwargs) -> None: ...
+    def warn(self, msg: str, **kwargs) -> None: ...
+    def err(self, msg: str, **kwargs) -> None: ...
+
+
+class StdLogger:
+    def info(self, msg: str, **kwargs) -> None:
+        sys.stdout.write(msg)
+
+    def warn(self, msg: str, **kwargs) -> None:
+        sys.stderr.write(msg)
+
+    def err(self, msg: str, **kwargs) -> None:
+        sys.stderr.write(msg)
```

Comment:
If you use just write, you won't have a newline after each msg.
Maybe we should use the print() function here.
```
print(msg, file=sys.stderr)
```